packages/wrangler/e2e/vitest.setup.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from "vitest";
+
+interface CustomMatchers {
+	toContainMatchingObject: (expected: object) => unknown;
+}
+
+declare module "vitest" {
+	interface Assertion extends CustomMatchers {}
+	interface AsymmetricMatchersContaining extends CustomMatchers {}
+}
+
+expect.extend({
+	// Extend vitest's expect object so that it has our new matcher
+	toContainMatchingObject(received: object[], expected: object) {
+		// const matcher = createMatcher(expected);
```

Comment:
Remove?

Reply:
Argh. Will do it in another PR to avoid going through CI hell again